### ISSUE TITLE
golang binaries for extensions

### ISSUE THREAD

**Issue by geekgonecrazy** [flathub/com.visualstudio.code#40]
I see flathub has `org.freedesktop.Sdk.Extension.golang` i've installed it.  Is there a way to make this flatpak utilize it?

https://dev.to/satimewallin/go--mess-with-flatpakvscode-ja6  Here is something someone has done to get around this.  Its definitely not ideal though :(

**Comment by stobbsm** [comment]
I've been working on this for myself as well. Haven't made much progress, as it seems the Extensions (golang, gcc8, openjdk) aren't having their PATH and other env variables used by the base vscode flatpak.

In the integrated terminal, I can find the tools as expected, but I can't seem to modify the PATH or provide other env overrides before it launches.

**Comment by stobbsm** [comment]
I got it working by overriding the PATH of com.visualstudio.code.

`flatpak override --env=PATH=/app/bin:/usr/bin:/usr/lib/sdk/golang/bin --env=GOROOT=/usr/lib/sdk/golang --env=GOPATH=$HOME/go`

Installed all tools from the go extension as expected that way.

**Comment by geekgonecrazy** [comment]
Does this just have to be ran just the one time?

**Comment by csoriano1618** [comment]
@geekgonecrazy yes, `flatpak override` creates a file with the overrides configuration in /var/lib/flatpak/overrides

**Comment by stobbsm** [comment]
We need to document this somewhere. Most of the time, it seems people stop using the flatpak options because they can't access their chains easily.

**Comment by jgttech** [comment]
I wanted to share this because some others might find this helpful. There are other threads about this but they are **_closed_**. I wanted to post this to an open thread that might actually be useful for folks. I was having serious trouble getting the Go lang IDE extension to work because it could not find the system go binary for the LSP to work. Here is a rundown of how I fixed it. This might help others trying to find any alternative to get things working with their IDE.

_**Solution**_
I could not get the Go lang LSP working with the IDE extension. I installed and used Flatseal to edit the "Environment" section of your IDE in Flatseal and added the environment configuration variables there. Closed the app, re-opened, and checked that it worked and it did.
